dragonchain/transaction_processor/level_5_actions.py

```diff
@@ -107,6 +109,22 @@ def execute() -> None:
     # Only check confirmations if there are any pending
     check_confirmations()
 
+    # go through the failed claims backlog
+    _log.info("Trying to finalize any previously failed claims...")
+    process_claims_backlog()
+
+
+def process_claims_backlog() -> None:
+    claims_set = redis.smembers_sync("mq:failed-claims")  # used sadd to make a set
```

Comment:
shouldn't be hard-coding strings like "mq:failed-claims". Make this as a constant in the module, or something

Reply:
Good point.  Fixed.